### PR TITLE
fix(banner): added method for deletion and sorting

### LINKED ISSUE #666: Error with deletion method for clickable banners
### What happened?

When deleting a clickable banner through postman or through frontend an error arises as no data is received.

image

### Which users are seeing the problem?

Superadmins (Index), Admins (HS)

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by callumgran**
## Proposed changes
Added method to delete banners so frontend does not get an error.
Banners are now sorted by visibility date and not creation.

Issue number: closes #666 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] The fixtures have been updated if needed (for migrations)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: app/communication/views/banner.py [hunk 1]
```diff
@@ -13,9 +13,13 @@
 class BannerViewSet(BaseViewSet):
     serializer_class = BannerSerializer
     pagination_class = BasePagination
-    queryset = Banner.objects.all()
+    queryset = Banner.objects.order_by("visible_from")
```
Comment (thomsen85) [inline app/communication/views/banner.py:16]: Ikke gjør dette er du grei. Bruk heller django filter og muligheten for frontend å velge ordering.

**Review by thomsen85 — DISMISSED** [review]

File: app/communication/views/banner.py (hunk 1 above)
Comment (thomsen85) [reply app/communication/views/banner.py:16]: Ser også at dette er feil endring for hva pr sier.

Comment (callumgran) [reply app/communication/views/banner.py:16]: Ja whoops, innser at jeg glemte å lage ny branch fra main.

**Comment by martcl** [comment]
Closing this on favor of the other one.

**Pull request closed without merging**